specs/darwin/es_authentication_events.table

```diff
@@ -0,0 +1,40 @@
+table_name("es_authentication_events")
+description("Logs authentication-related events from EndpointSecurity.")
+schema([
+    Column("time", BIGINT, "Time of event in UNIX seconds"),
+    Column("version", INTEGER, "Version of EndpointSecurity event"),
+    Column("seq_num", BIGINT, "Per event sequence number"),
+    Column("global_seq_num", BIGINT, "Global sequence number"),
+    Column("event_type", TEXT, "Type of authentication event (e.g., 'authentication', 'openssh_login', 'sudo')"),
+    Column("pid", BIGINT, "Process ID of the authenticating process"),
+    Column("pidversion", INTEGER, "Process ID version"),
+    Column("path", TEXT, "Path of the executable involved in the authentication"),
+    Column("username", TEXT, "Username associated with the authenticating process"),
+    Column("description", TEXT, "Human-readable description of the event"),
+    Column("category", TEXT, "Event category (always 'authentication' for this table)"),
+    Column("severity", TEXT, "Severity level of the event (low, medium, high)"),
+    Column("success", INTEGER, "Indicates if the authentication attempt was successful (1) or not (0)"),
+    Column("auth_type", TEXT, "Authentication mechanism used (e.g., 'password', 'touchid', 'ssh')"),
+    Column("result_type", TEXT, "Detailed result of the authentication attempt"),
+    Column("auth_right", TEXT, "Authorization right being checked (for authorization events)"),
+    Column("remote_address", TEXT, "Remote IP address (for SSH/screensharing events)"),
+    Column("remote_port", INTEGER, "Remote port (for network-related auth events)"),
+    Column("ssh_login_username", TEXT, "Username for SSH login (may differ from process username)"),
+    Column("su_from_username", TEXT, "Original username for su events"),
+    Column("su_to_username", TEXT, "Target username for su events"),
+    Column("sudo_command", TEXT, "Command executed with sudo"),
```

Comment:
on one hand, I can see why you might want one authentication events table. On the other hand, this feels really unrelated and messy